src/find/matchers/exec.rs

```diff
@@ -83,6 +86,25 @@ impl Matcher for SingleExecMatcher {
                 }
             }
         }
+
+        // support interactive exec
+        if self.interactive {
+            let tips = format!(
+                "{} ... {} > ? [y/n]: ",
+                self.executable,
+                path_to_file.to_string_lossy()
+            );
+            #[allow(clippy::explicit_write)]
+            write!(stdout(), "{}", tips).unwrap();
```

Comment:
POSIX [specifies](https://pubs.opengroup.org/onlinepubs/9799919799/utilities/find.html#tag_20_47_11)

- The -ok primary shall write a prompt to *standard error*
- In the POSIX locale, the last non-&lt;blank> in the prompt shall be '?'

Reply:
Thanks! I overlooked these, I will fix them later.